website/docs/r/firewall.html.markdown

```diff
@@ -76,20 +78,34 @@ The following arguments are supported:
 
 ---
 
-A `ip_configuration` block supports the following:
+An `ip_configuration` block supports the following:
 
 * `name` - (Required) Specifies the name of the IP Configuration.
 
 * `subnet_id` - (Optional) Reference to the subnet associated with the IP Configuration.
 
--> **NOTE** The Subnet used for the Firewall must have the name `AzureFirewallSubnet` and the subnet mask must be at least `/26`.
+-> **NOTE** The Subnet used for the Firewall must have the name `AzureFirewallSubnet` and the subnet mask must be at least a `/26`.
 
 -> **NOTE** At least one and only one `ip_configuration` block may contain a `subnet_id`.
 
 * `public_ip_address_id` - (Required) The ID of the Public IP Address associated with the firewall.
 
 -> **NOTE** The Public IP must have a `Static` allocation and `Standard` sku.
 
+---
+
+A `management_ip_configuration` block supports the following:
+
+* `name` - (Required) Specifies the name of the IP Configuration.
+
+* `subnet_id` - (Required) Reference to the subnet associated with the IP Configuration.
```

Comment:
We should note the ForceNew behavior on the `subnet_id`. It would also be good to add this to the same property in the `ip_configuration` block.

```suggestion
* `subnet_id` - (Required) Reference to the subnet associated with the IP Configuration. Changing this forces a new resource to be created.
```